### PR TITLE
Redid campus events page

### PR DESCRIPTION
**Pull request by Christiaanvdm**
NewGIF

### CODE REVIEW

File: django_project/core/base_static/css/stylesheet.css [hunk 1]
```diff
@@ -835,3 +842,7 @@ nav#sidebar ul li a:hover {
     background-color: var(--c-smoke);
 }
 
+.input-invalid {
+    border: 2px dashed red !important;
```
Comment (pierrealixt) [inline django_project/core/base_static/css/stylesheet.css]: are we keeping the dashed border?

File: django_project/ford3/migrations/0045_campusevent_deleted.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.7 on 2019-05-16 18:19
```
Comment (pierrealixt) [inline django_project/ford3/migrations/0045_campusevent_deleted.py:1]: this file should not be here.

File: django_project/ford3/views/campus_events.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+from django.http import HttpResponse
+from django.core.exceptions import ValidationError
+from ford3.models.campus_event import CampusEvent
+from ford3.models.campus import Campus
+
+
+def create_or_update(request, campus_id):
+
+    if 'id' in request.POST:
+        # We are updating
```
Comment (pierrealixt) [inline django_project/ford3/views/campus_events.py]: remove the comment

File: django_project/ford3/migrations/0045_campusevent_deleted.py (hunk 2 above)
Comment (pierrealixt) [reply django_project/ford3/migrations/0045_campusevent_deleted.py:1]: oh, I thought it was for the Campus model.

**Comment by pierrealixt** [comment]
Screen Shot 2019-05-17 at 11 56 31

Screen Shot 2019-05-17 at 12 10 47

**Comment by pierrealixt** [comment]
Screen Shot 2019-05-17 at 11 57 05
padding/margin is weird here

File: django_project/ford3/templates/campus_form.html [hunk 4]
```diff
@@ -44,6 +44,135 @@ <h2>{{ campus }} Campus - {{ form_name_list|index:wizard.steps.step0 }}</h2>
...
+
+                                    <div id='form-event'>
+                                        <label>Name*</label>
+                                        <input required placeholder="eg. Open day" type='text' name='name' />
+                                        <label>Start date*</label>
+                                        <input required placeholder="2019-10-26" type='text' name='date_start' class='mydatepicker' />
+                                        <label>End date*</label>
+                                        <input placeholder="2019-10-28" required type='text' name='date_end' class='mydatepicker' />
+                                        <label>Website</label>
+                                        <input type='text' placeholder="http://www.SiteWithEventInformation.co.za" name='http_link' />
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus_form.html]: just use lowercase

**Comment by pierrealixt** [comment]
bugeventurl

**Comment by pierrealixt** [comment]
`<hr />` is missing and margin/padding is weird.

Screen Shot 2019-05-17 at 12 00 56

**Comment by pierrealixt** [comment]
It should order by start_date asc

**Comment by pierrealixt** [comment]
I deleted all my events but the `<hr />` are still present
Screen Shot 2019-05-17 at 12 03 26

Instead of `hr`, you can use the class `border-bottom` to create the same effect.

**Comment by pierrealixt** [comment]
Maybe something that looks better?
Look in the styleguide
Screen Shot 2019-05-17 at 12 04 36

**Comment by pierrealixt** [comment]
The error message needs a bit of styling
Screen Shot 2019-05-17 at 12 05 25

**Comment by pierrealixt** [comment]
bugeventurl-2

**Comment by pierrealixt** [comment]
Rebase your PR, I have fixed the build.

File: django_project/core/base_static/css/stylesheet.css (hunk 1 above)
Comment (Christiaanvdm) [reply django_project/core/base_static/css/stylesheet.css]: I thought this was the plan?

Comment (pierrealixt) [reply django_project/core/base_static/css/stylesheet.css]: that was the plan when we were in dev mode. 
Do you think it looks good?

**Comment by Christiaanvdm** [comment]
NewGIF

**Review by pierrealixt — APPROVED** [review]
LGTM

**Pull request merged**